src/Jobs/OptimizeHtml.php

```diff
@@ -1,25 +1,37 @@
 <?php
 
-namespace SiteOrigin\PageCache\Listeners;
+namespace SiteOrigin\PageCache\Jobs;
 
+use Illuminate\Bus\Queueable;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
 use Illuminate\Support\Collection;
 use SiteOrigin\PageCache\Events\PageOptimized;
 use SiteOrigin\PageCache\Events\PageOptimizing;
-use SiteOrigin\PageCache\Events\PageRefreshed;
 use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Contracts\Queue\ShouldBeUniqueUntilProcessing;
+use SiteOrigin\PageCache\Page;
 
-class OptimizeHtml implements ShouldQueue
+class OptimizeHtml implements ShouldQueue, ShouldBeUniqueUntilProcessing
```

Comment:
Should we be using `ShouldBeUniqueUntilProcessing`, or `ShouldBeUnique`. Technically, there's no good reason for another version of this job to be queued while it's busy processing. So this slightly reduces any edge cases that might result in a duplicate job.

Reply:
This does not seem to stop specific jobs with a different uniqueId